File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        if(s.size()!=t.size()) return false;
        int i = 0;
        for(char p:s){
            long long idx = t.find(p);
            if(idx==string::npos) return false;
            t.erase(idx,1);
            i++;
        }
        return true;
    }
};

File: 0242-valid-anagram/0242-valid-anagram.py
class Solution(object):
    def isAnagram(self, s, t):
        dict1 = {}
        dict2 = {}
        for i in s:
            dict1[i] = dict1.get(i,0)+1
        for i in t:
            dict2[i] = dict2.get(i,0)+1
        if(dict1==dict2):
            return True
        else:
            return False 